UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs

```diff
@@ -261,7 +261,7 @@ public void ResitOthers()
 				}
 				else
 				{
-					spriteHandler.ChangeSpriteVariant(0, false);
+					spriteHandler.ChangeSpriteVariant(spriteHandler.InitialVariantIndex, false);
```

Comment:
initial variant index is too confusing

Reply:
also I have changes here in the PR

Reply:
but thats what the default should be??

Reply:
well, it only runs it if it's not being set Rotational

Reply:
if (ChangeSprites == false)

Reply:
ChangeSprites sounds like a method, not a bool

Reply:
Done, back to 0